tests/rptest/services/redpanda.py

```diff
@@ -2448,11 +2448,13 @@ def __init__(self,
                     self.LOG_LEVEL_KEY, self.DEFAULT_LOG_LEVEL)
             else:
                 self._log_level = log_level
-            self._log_config = LoggingConfig(self._log_level, {
-                'exception': 'debug',
-                'io': 'debug',
-                'seastar_memory': 'debug'
-            })
+            self._log_config = LoggingConfig(
+                self._log_level, {
+                    'exception': 'debug',
+                    'io': 'debug',
+                    'seastar_memory': 'debug',
+                    'dns_resolver': 'info'
```

Comment:
Did you also mean to add storage-resources here?

Reply:
i would love to but Redpanda do not start when a logger is unavailable (all the upgrade tests fails), i was thinking about backporting that change at least to v23.3.x and then enabling it in v24.2.x and dev

Reply:
Ahh got it. Backporting makes sense to me. I also considered maybe annotating upgrade tests with some specific logging config, but maybe it's more effort than it's worth